l1-contracts/scripts/register-hyperchain.ts

```diff
@@ -97,12 +97,12 @@ async function main() {
       await checkTokenAddress(baseTokenAddress);
       console.log(`Using base token address: ${baseTokenAddress}`);
 
-      const useGovernance = !!cmd.useGovernance && cmd.useGovernance === "true";
```

Comment:
a more nit approach here is just have `--use-governance` as a CLI param (without the `< use-governance >`). This will be treated as a boolean flag and will always have boolean type